ansible/roles/nomad-client/templates/40-client.hcl

```diff
@@ -33,6 +33,14 @@ client {
     read_only = {{volume.read_only|bool|lower}}
   }
 {% endfor %}
+
```

Comment:
This newline should go inside the conditional block so that it doesn't result in a newline on all machines.